website/docusaurus.config.ts

```diff
@@ -475,9 +475,9 @@ export default async function createConfigAsync() {
         respectPrefersColorScheme: true,
       },
       announcementBar: {
-        id: 'announcementBar-3', // Increment on change
+        id: 'announcementBar-4', // Increment on change
```

Comment:
No need to increment; this is just a typo

```suggestion
        id: 'announcementBar-3', // Increment on change
```